### PR TITLE
Refactor VTK config key and add possible VTK quantities

### PR DESCRIPTION
**Pull request by RobinSattler**
Since I'm working on animations for the smash-vhlle-hybrid, I added a few possible VTK quantities and improved the source code a bit which I describe more detailed below.
 
### What I did
- Refactored the VTK config keys: Removed unnecessary `VTK output` config key. Now only the desired VTK output value has to be given in the config by using `VTK_output_values quantity` (e.g., `VTK_output_values eps`).
- Improved structure of VTK source code files vtk.cpp and vtk.h w.r.t. readability and added a check if quantities given in the config file are implemented VTK quantities. Prior to this change, for each invalid quantity a VTK output file was produced which only contained the information of the function write_header. The check if a quantity is valid prevents this creation of basically empty files.
- Implemented the bulk pressure (Pi), the shear stress tensor (pi), and the velocity (v) as options for VTK output.
- Removed trailing whitespaces in the files I worked with and fixed a few typos in the README.txt file.
 
### What I tested
- Created VTK output for `eps`, `nb`, `nq`, `ns`, and `p` with my changes and with the `stable_ebe` branch (with the same config and IC) and compared the outputs. They were identical, so my changes shouldn't have changed anything output-wise for the already implemented quantities.
- Animated a few of the quantities which all seemed reasonable and I didn't spot anything weird.
- I also looked at the implemented Cartesian output and it seems very buggy to me, but I'll investigate that over the next couple of weeks and might open an issue to start a discussion about it.
 
Since I am not a contributor listed in the repo of vhlle, I cannot request a review from anybody. @NGoetz, could you assign yourself (since you implemented VTK output in the first place) and @yukarpenko?

### CODE REVIEW

**Comment by NGoetz** [comment]
I somehow can't assign reviewers, but I can assign people instead.

File: src/vtk.cpp [hunk 1]
```diff
@@ -119,24 +222,44 @@ std::vector<std::string> split (const std::string &s, char delim) {
...
+    } else if (valid_quantities_.at(q) == "tensor") {
+      write_vtk_tensor(file, h, q);
+    } else {
+      std::cout << "Quantity '" << q << "' is neither stated to be a scalar, "
+        "nor a vector, nor a tensor. Skipping this quantity. Please check the "
+        "map in file src/vtk.h." << std::endl;
+    }
   }
-  
+
```
Comment (NGoetz) [inline src/vtk.cpp:262]: There should be probably also a file close here.

File: src/vtk.cpp [hunk 2]
```diff
@@ -37,79 +38,181 @@ std::string VtkOutput::make_filename(const std::string &descr, int counter) {
...
-        pos=total_length/(z_length/2)*(iz-z_length/2);
-        for (int ieta=h.getFluid()->getNZ()/2; ieta<h.getFluid()->getNZ() ; ieta++){
-        if(h.getTau()*std::sinh(h.getFluid()->getDz()*(ieta-h.getFluid()->getNZ()/2))>pos){
-          factor=fabs((h.getTau()*std::sinh(h.getFluid()->getDz()*(ieta-h.getFluid()->getNZ()/2))-pos)/pos);
-          poseta=ieta;
-          break;
+        cell->getPrimVar(eos_, h.getTau(), e, p, nb, nq, ns, vx, vy, vz);
+        cell2->getPrimVar(eos_, h.getTau(), e2, p2, nb2, nq2, ns2, vx2, vy2, vz2);
+        double q = 0;
+        if (quantity == "eps") {
```
Comment (NGoetz) [inline src/vtk.cpp:103]: If we are anyway at it, this could deserve some reduction of code duplication.

**Review by NGoetz — APPROVED** [review]
Looks good to me in general, I just have two suggestions. I nevertheless approve already.

File: src/vtk.cpp (hunk 1 above)
Comment (RobinSattler) [reply src/vtk.cpp:263]: True, done.

**Comment by yukarpenko** [comment]
Many thanks for correcting all the typos in Readme - I appreciate it! Most of the changes are in the VTK output, which is something I never used (not with the manpower I have), for which my only hope is that it works OK and does not hit the rest of the code with unsafe memory access :) So I am just merging as is.
Which got me thinking that the code as a whole would certainly benefit from some sort of re-factoring, but it is something I couldn't do due to the reason I mentioned...

**Pull request merged**